src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -328,7 +388,6 @@ do_read()
     impl().ws_.async_read(op_, rb_, strand_.wrap(
         std::bind(&BaseWSPeer::on_read,
             impl().shared_from_this(), placeholders::error)));
-    cancel_timer();
```

Comment:
According to David the answer is yes. Resetting the timer on normal reads or writes can have counterintuitive results. @JoelKatz ?